### PR TITLE
Fundraising Thank You email

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: en-US/emails/2017/fundraising_thank_you.lang [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+;Cute Thank You:
+Cute Thank You:
+
+
+;Click on whichever thank you <em>you</em> like (and you know what, if you want to click all three and be thanked more than once, be our guest, you deserve it!)
+Click on whichever thank you <em>you</em> like (and you know what, if you want to click all three and be thanked more than once, be our guest, you deserve it!)
+
+
+;Stay tuned for much more to come in 2017. It’s going to be a big year.
+;Stay tuned for much more to come in 2017. It’s going to be a big year.
```
Comment (flodolo) [inline en-US/emails/2017/fundraising_thank_you.lang]: Double reference string

File: en-US/emails/2017/fundraising_thank_you.lang [hunk 2]
```diff
@@ -0,0 +1,58 @@
+## NOTE: Thank you email for the 2016 fundraising campaign. Email screenshot: https://cloud.githubusercontent.com/assets/12228435/21836541/cfde85d2-d77a-11e6-8566-4d28f0667606.png
+
+
+# Email subject
+;Thanks!
+Thanks!
+
+
+# Preheader (only displayed in thread view https://www.campaignmonitor.com/assets/uploads/2015/08/preheader-text-on-mobile.png ), try to keep the translation around 40-50 characters, you can rephrase as needed.
```
Comment (TheoChevalier) [inline en-US/emails/2017/fundraising_thank_you.lang]: old habit just to make sure the parenthesis is not considered part of the link, will remove it

File: en-US/emails/2017/fundraising_thank_you.lang (hunk 1 above)
Comment (TheoChevalier) [inline en-US/emails/2017/fundraising_thank_you.lang]: gha.

File: en-US/emails/2017/fundraising_thank_you.lang [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+;Sincere Thank You:
+Sincere Thank You:
+
+
+# Title associated to a puppy emoji
+;Cute Thank You:
+Cute Thank You:
+
+
+;Click on whichever thank you <em>you</em> like (and you know what, if you want to click all three and be thanked more than once, be our guest, you deserve it!)
```
Comment (TheoChevalier) [inline en-US/emails/2017/fundraising_thank_you.lang]: technically it’s correct, but I agree it’s not the most elegant phrasing…

Comment (flodolo) [inline en-US/emails/2017/fundraising_thank_you.lang]: Ah, now I get it. Horrible but it makes sense.

**Comment by TheoChevalier** [comment]
I’ll have to put this one on hold, just got notified that a sentence will be added

**Pull request merged**